docs/useful-libraries/react/README.md

```diff
@@ -2,25 +2,25 @@
 
 ## General
 
-* [Reach UI](https://reacttraining.com/reach-ui/)
+* [Reach UI](https://reacttraining.com/reach-ui/) - Accessible foundation for React apps
```

Comment:
Again I would be tempted to be more specific here, by giving examples of specific things that Reach UI has/does